src/main/java/seedu/address/model/Transcript.java

```diff
@@ -171,8 +178,8 @@ public String toString() {
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Transcript // instanceof handles nulls
-                && modules.equals(((Transcript) other).modules));
+            || (other instanceof Transcript // instanceof handles nulls
```

Comment:
I see that you removed 1 level of indent but for codes that are wrapped-around or continuation indentation, you should be using 2 level of indent.

Reply:
Sorry, I think I enabled reformat on save. I will try and find the rule and change it.

Reply:
Done! Changed editor code reformat setting -  for those curious: Editor/Code Style/Java/Tabs and Indents/Continuation Indent = 8.